### ISSUE TITLE
Bring mrnaseq instructions up to date (under mrnaseq/)

### ISSUE THREAD

**Issue by ctb** [ged-lab/khmer-protocols#151]
- [ ] change from m1.xlarge
- [ ] fix nemo => nema (nematostella)
- [ ] check with @mr-c - is the Debian URL for the adapters stable? (https://sources.debian.net/data/main/t/trimmomatic/0.32+dfsg-2/adapters/TruSeq3-PE.fa)
- [ ] eliminate the use of ${HOME} as on Amazon computers this is too small; OR, mount something there.
- [ ] what does %% do in for loops?
- [ ] remove fastx stuff
- [ ] remove THREADS from filter-abund command as it's not any faster.
- [ ] question @mr-c as to the switch to using multiple directories for the data; this will confuse many people

**Comment by mr-c** [comment]
Yes, that URL is stable. I'll file a bug for the adapters to be included in future packages.

`%%` inside a shell parameter specification causes the trimming from the end of the parameter value of what follows the `%%`. Just another way to swap out filename extensions; mostly equivalent to your use of basename.

Docs at http://www.gnu.org/software/bash/manual/bash.html#Shell-Parameter-Expansion

I'm very surprised to hear that you think using a subdirectory for each stage of processing will confuse people. I find dozens of files whose names differ by just a few characters in one big directory to be confusing.

**Comment by mr-c** [comment]
Trimmomatic bug is filed: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=779296

**Comment by ctb** [comment]
On Thu, Feb 26, 2015 at 09:54:21AM -0800, Michael R. Crusoe wrote:
> I'm very surprised to hear that you think using a subdirectory for each stage of processing will confuse people. I find dozens of files whose names differ by just a few characters in one big directory to be confusing. 

Paths/directorys are #1 confusion for people new to the shell, we find.
